### DISCUSSION TITLE
crashes on startup

### DISCUSSION THREAD

**Discussion by Tomcatnet** [hassio-addons/addon-unifi#260] [General]
when i start the controller all home assistant hangs, and I have to restart the raspberry

**Comment by duckyzcy** [comment]
i have the same as well... The entire homeassistant crashes when i start unify-addon.
I used to have unify addon on 32bit and everything was working fine. since version 1.0 is requiring 64bit i migrate my system to 64bit on RPI3.
Even a fresh installation of unify-addon will crash the entire system after a couple of minutes of starting the addon.

**Comment by eisdieledk** [comment]
I have the Same Problem. RPI3 on 64 Bit. As soon as I start the AddOn it crashes Everything. I can only hardreset the RPI Boy unplugging. 
I tried with 2.2.2 and 2.3.0, and also just updated HA to the latest version. First I believed maybe a TCP Port issue, but doublechecked all used ports and can‘t see a Problem. Also I got it working for a short time (5 Minuten maybe) than it crashed again. I‘m happy to Share logfiles or other Details, just let me know where to find it.

**Reply by frenck** [reply]
> Also I got it working for a short time (5 Minuten maybe) than it crashed again

This is generally an indicator your system runs out of memory. Please note, that an RPi3 only has 1Gb of memory for everything. That is not a lot.

**Reply by eisdieledk** [reply]
Thanks for the fast reply. This was the next idea I had. Will try with a RPi4 next. Thank you again!
